### ISSUE TITLE
WordPress 5.0 Appears Incompatible

### ISSUE THREAD

**Issue by LordA98** [RadoslavGeorgiev/ultimate-fields#25]
Updated to WordPress 5.0 and the Post Types don't appear to work anymore.  My containers are set to appear certain pages, which works for WordPress 4.9.  Upgrading to 5.0 seems to ignore the Post Types rules that I have set - now all containers are appearing on all pages.  Fortunately, I reverted to the previous version and it's working again.

**Comment by aimanpathan** [comment]
Yes, same happened with me, kindly let me know if anyone has any solution for the same.
